src/lib/frontend/frontend.ml

```diff
@@ -39,11 +39,11 @@ let unused_context name context =
   | None -> false
   | Some s -> not (Util.SS.mem name s)
 
-type 'a status =
+type status =
   | Unsat of Commands.sat_tdecl * Ex.t
   | Inconsistent of Commands.sat_tdecl
-  | Sat of Commands.sat_tdecl * 'a
-  | Unknown of Commands.sat_tdecl * 'a
+  | Sat of Commands.sat_tdecl
+  | Unknown of Commands.sat_tdecl
```

Comment:
I am slightly surprised that we were never using these environments, but if that is the case, there is no point to keep them anyways.

Reply:
Me too. Our API is such an onion with too many layers, we didn't notice it before.